src/sass/components/_avatar.scss

```diff
@@ -18,3 +18,25 @@
 		z-index: 10;
 	}
 }
+
+.about-avatar {
```

Comment:
Have we examined how this rule is applied in small-screen devices?

Reply:
Examined and added the gutters.